.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 Cargo.lock
+.idea
```

Comment:
It's generally good practice to avoid putting IDE-specific ignores in a project's `.gitignore`, because it's not feasible to list every file generated by every IDE. Instead, put the ones for IDEs you use in your [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)

Reply:
Oh, I think I added that on accident, I'll just get rid of that